web/src/main/java/com/bazaarvoice/emodb/web/scanner/writer/KafkaScanDestinationWriter.java

```diff
@@ -31,7 +31,7 @@
     private final BlockingQueue<CoordinateAndFuture> _futureQueue;
     private final ExecutorService _futureGettingService;
     private final AtomicReference<Throwable> _error = new AtomicReference<>();
-    private boolean _closed;
+    private volatile boolean _closed;
```

Comment:
do multiple threads access the same instance of a `KafkaScanDestinationWriter`? As you know, there's a nontrivial cost to using `volatile` -- if this variable is read frequently, say on every iteration, it could add up to a lot of total time. Is there an alternative, like running an independent writer for each thread, or do they need to be synchronized?